frontend/src/metabase/query_builder/components/chart-type-selector/use-chart-type-visualizations.ts

```diff
@@ -14,29 +14,38 @@ import {
 import { DEFAULT_VIZ_ORDER } from "./viz-order";
 
 export type UseChartTypeVisualizationsProps = {
-  question: Question;
+  question?: Question;
   onUpdateQuestion: (question: Question) => void;
+} & GetSensibleVisualizationsProps;
+
+export type UserChartVisualizationsReturn = {
```

Comment:
nit: manually typing the return values like this is a bit verbose and un-necessary in my opinion, I see that without it we don't see "CardDisplayType" but `"table" | ...` but I would rather fix the issue upstream instead of having make the code verbose in more places.

Let me try something quick

Reply:
Ok it's apparently impossible to make it show "CarDisplayType" because of the way the type is defined, I think we're ok with just returning the "aaa" | "bbb"  as it's still type safe though, what do you think @oisincoveney?